src/instanceMethods/buttons-handlers.js

```diff
@@ -88,5 +88,4 @@ const getInputValue = (instance, innerParams) => {
 const getCheckboxValue = (input) => input.checked ? 1 : 0
 
 const getRadioValue = (input) => input.checked ? input.value : null
-
-const getFileValue = (input) => input.files.length ? input.files[0] : null
+const getFileValue = (input) => input.getAttribute('multiple') ? (input.files.length ? input.files : null) : (input.files.length ? input.files[0] : null)
```

Comment:
I believe this will be shorted, more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and also more readable:

```js
const getFileValue = (input) => input.files.length ? (input.getAttribute('multiple') ? input.files : input.files[0]) : null
```

Reply:
I did not even think of such an option, how simple and concise, thank you